docassemble/ALKilnInThePlayground/data/static/alkiln_playground.css

```
@@ -14,20 +70,54 @@ body {
 #alkiln_test_output h3 {
   font-size: 1.2em
 }
+#alkiln_test_output .accordion-header h3 {
+  font-size: 1.1em;
+}
+#alkiln_test_output .accordion-header h3 {
+  font-size: 1.1em;
+}
+
+:root {
+  --accordion-header-edge: calc(1.5em * .5);
+}
+
+#alkiln_test_output .accordion-header h3 {
+  padding-left: var(--accordion-header-edge);
+}
```

Name for clarity, but maybe a note would be better. I think we're not reusing this at the moment.

```suggestion
#alkiln_test_output .accordion-header h3 {
  note: "Based on previous hard-coded accordion styles";
  padding-left: calc(1.5em * .5);
}
```